.github/workflows/test_models_net_tests.yml

```diff
@@ -48,6 +48,14 @@ jobs:
         ]
         dotnet-version: [ '6.0.x' ]
         os: [ ubuntu-latest ]
+        dafny-version: ${{ inputs.dafny }}
+        exclude:
+          # ExternV2 is only supported on Dafny >4.4.0 (unreleased; only on nightly as of commit)
```

Comment:
@seebees I thought you said Dafny had already released this?
If Dafny has not released this, how can we use it for the ACCP work?